bundles/org.openhab.binding.mihome/src/main/resources/OH-INF/config/config.xml

```diff
@@ -36,6 +36,14 @@
 			<advanced>true</advanced>
 		</parameter>
 
+		<parameter name="interface" type="text">
+			<context>network-address</context>
```

Comment:
The `network-address` context validates hostnames and IP adresses. AFAIK, there is no context for interface names. So, this should be removed.

Reply:
Removed. Unfortunately, I'm not accustomed with the effect of the "contexts", so I blindly replicated based on the other fields.